src/Test/L0/Worker/Build/TrackingManagerL0.cs

```diff
@@ -520,5 +520,12 @@ public void UpdatesTrackingConfigJobRunProperties()
                 Assert.True(DateTimeOffset.Now.AddSeconds(1) >= config.LastRunOn);
             }
         }
+
+        //removes CR so the test can run on Linux, even if it was synced on a Windows drive share
```

Comment:
This seems like a non-goal.  The requirement to build on linux is to sync on linux.  I concur that I don't want code to encourage this pattern.  

I don't think we should take this.  Sync on your linux box.